### DISCUSSION TITLE
Wagtail tutorials

### DISCUSSION THREAD

**Discussion by paragliding-nsk** [torchbox/wagtail-grapple#358] [Q&A] [answered]
Hello, I have one more "newbie" question: what courses are the best for learning Wagtail 5? I found this one:
https://www.youtube.com/playlist?list=PLMQHMcNi6ocsS8Bfnuy_IDgJ4bHRRrvub
 but it's quite old and I'm not sure - is it still actual?
Or this one:
https://www.youtube.com/watch?v=elEJRh4Xg-g&list=PLeLGN29skXLqWeK-maeiDwVSUH1q11-4T
it's newer, but not as detailed.

**Comment by zerolab** [comment, marked as answer]
Best to ask in the Wagtail Slack (in the #support channel). More details https://github.com/wagtail/wagtail/wiki/Slack

I would start with https://docs.wagtail.org/en/stable/getting_started/index.html. Once one with the basic tutorial there, and have read more of the docs, I'd then look at the various tutorials. Most changes would be in imports (e.g. `wagtail.core.something` became `wagtail.something`)

**Reply by paragliding-nsk** [reply]
Thanks a lot

**Reply by zerolab** [reply]
Also, I suggest using https://github.com/wagtail/wagtail/discussions for any Wagtail-related topics, rather than here ;)
